lib/urlwatch/filters.py

```diff
@@ -183,6 +183,20 @@ def filter(self, data, subfilter=None):
         return ical2text(data)
 
 
+class JsonFormatFilter(FilterBase):
+    """Convert to formatted json"""
+
+    __kind__ = 'formatted-json'
+
+    def filter(self, data, subfilter=None):
+        from json import dumps
+        from json import loads
+        parsed_json = loads(data)
+        if subfilter is not None:
```

Comment:
That's kind of undocumented, wonder if it would make sense to configure e.g. indentation with the subfilter instead, and make the getting some keyed item from JSON a separate filter instead of tacking it onto the "formatting" filter (IMHO it might be counter intuitive that formatting also "processes" the JSON).

Reply:
Great idea. I didn't really like the idea of filtering one item from the json especially as it only allows filtering it from the root level. I changed it now to allow specifying the indentation level.